aurorastation.dme

```diff
@@ -1271,7 +1270,6 @@
 #include "code\modules\clothing\rings\rings.dm"
 #include "code\modules\clothing\shoes\colour.dm"
 #include "code\modules\clothing\shoes\jobs.dm"
-#include "code\modules\clothing\shoes\leg_guard.dm"
```

Comment:
Why is this being removed?

Reply:
This code is from a previous PR, please put it back.

Reply:
So very sorry